neo/rawio/neuralynxrawio.py

```diff
@@ -148,11 +148,11 @@ def _parse_header(self):
 
                     if (os.path.getsize(filename)<=HEADER_SIZE):
                         self._empty_nse_ntt.append(filename)
-                        data = dict(unit_id    = np.array([]), 
-                                    channel_id = np.array([]), 
-                                    timestamp  = np.array([]), 
-                                    features   = np.array([[]]), 
-                                    samples    = np.array([[]]))
+                        data = dict(unit_id    = np.zeros((0,), dtype=dtype), 
+                                    channel_id = np.zeros((0,), dtype=dtype), 
+                                    timestamp  = np.zeros((0,), dtype=dtype), 
+                                    features   = np.zeros((0,0), dtype=dtype), 
+                                    samples    = np.zeros((0,0), dtype=dtype))
```

Comment:
Oups.
My proposal was:
```data = np.zeros((0,), dtype=dtype)```
would replace all lines 151 - 155
data must be numpy .array 1D  (memmap or not) with a structured dtype not a dict.

Reply:
done)